### ISSUE TITLE
[Remediate] Keyboard navigation for Item Tools dropdown

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/dacs_handbook#323]
### Steps to recreate
1. Login to ARES.
2. Select the "Library Sandbox" Course.
3. Select the option "Reserves" from the course menu.
4. Switch to the Instructor mode.

### URL
[Canvas](https://canvas.princeton.edu/)

### Issues
Item Tools doropdown menu not accessible via keyboard

### Severity
Critical

### WCAG Conformance level (A, AA, AAA)
A

### User impact
Users cannot add an item or review usage unless they are using a mouse or equivalent device.

### Recommended fix
Add the elements to the tab index or change them to an element that is on the tab index.

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes

Per a scan done by John Jameson:

"As I have mentioned earlier, our colleague, John Jameson, conducted an accessibility testing of the Library Resources in Canvas.
Here is the issue he reported in his testing:

The menu students need to open to get to the link to access reserve materials cannot be operated by any assistive devices. It can only be opened by hovering over it with a mouse.

Suggested change: Rebuild the menu toggles using <button> tags with aria-expanded attributes, or shim the missing functionality using role/tabindex attributes and JS. Ref: [https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/button_role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/button_role)"

**Comment by kevinreiss** [comment]
Open support ticket with Atlas asking if we can run a second set of LTI pages that can be integrated with our Princeton Test Canvas to test a fix outside of the production system.

**Comment by kevinreiss** [comment]
Put in temporary fix: https://github.com/AtlasSystems/hosting-ares-princeton/pull/4.
